pkg/oc/cli/cmd/importimage.go

```diff
@@ -359,11 +359,6 @@ func (o *ImportImageOptions) createImageImport() (*imageapi.ImageStream, *imagea
 		return stream, isi, nil
 	}
 
-	// the stream already exists
-	if len(stream.Spec.DockerImageRepository) == 0 && len(stream.Spec.Tags) == 0 {
-		return nil, nil, fmt.Errorf("image stream does not have valid docker images to be imported")
-	}
-
```

Comment:
throwing this error seems correct. I'm confused by the issue the users opened, need to get some clarification on what they are expecting and why.

Reply:
@bparees Empty imagestreams can exist. This is not something wrong. You can remove all imagestream tags from imagestream and import new tags.

The `stream.Spec.DockerImageRepository` and `stream.Spec.Tags` are processed later in the `importAll` and `importTag`. So this check looks completely wrong for me.